website/content/docs/v0.7/Bare Metal Platforms/digital-rebar.md

```diff
@@ -80,7 +80,7 @@ We have some Digital Rebar [example files](https://github.com/talos-systems/talo
 To apply these configs you need to create them, and then apply them as follow:
 
 ```bash
-$ drpcli bootenvs create talos-0.7
+$ drpcli bootenvs create talos-0.6
```

Comment:
probably here we have more unintentional 0.6 references.

Reply:
:+1: Thank you.